CMakeLists.txt

```diff
@@ -35,11 +35,17 @@ add_compile_definitions(_GLIBCXX_USE_CXX11_ABI=0)
 if(NOT PYBIND11_DIR)
   set(PYBIND11_DIR pybind11)
 endif()
```

Comment:
```suggestion
```

We can remove these lines because we don't need to git-clone `pybind11` anymore.